server/service/apple_mdm.go

```diff
@@ -1052,6 +1052,7 @@ func (uploadAppleInstallerRequest) DecodeRequest(ctx context.Context, r *http.Re
 
 func (r uploadAppleInstallerResponse) error() error { return r.Err }
 
+// Deprecated: Not in Use
```

Comment:
Note aside: We should eventually remove these endpoints that were made for the MDM PoC years ago IIRC.